lib/Pusher.php

```diff
@@ -20,9 +20,9 @@
 		+ Phil Leggetter (phil@leggetter.co.uk)
 */
 
-class PusherException extends Exception
-{
-}
+require_once('PusherException.php');
+require_once('PusherHTTPException.php');
+require_once('TriggerResult.php');
```

Comment:
It would be better to rely on autoloading rather than having require_once calls (hint: Composer can handle the autoloading for you).
If you want to support easily people not using composer, you could ship a separate file registering an autoloader for your classes (most packages don't do it anymore though as such people will probably be using their own autoloader anyway)